package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "com.unity.webrtc",
   "displayName": "WebRTC",
-  "version": "3.0.0-pre.1",
+  "version": "3.0.0-pre.2",
   "unity": "2019.4",
```

I think this is the lowest version, so it will be 2020.3.
```suggestion
  "unity": "2020.3",
```